src/vault/client/http.clj

```diff
@@ -124,7 +124,7 @@
 
 ;; ## Authentication Methods
 
-(defn- api-auth!
+(defn api-auth!
```

Comment:
Maybe mark this with `^:no-doc`? It could use a docstring for source readers though.

Reply:
TIL about the `:no-doc` tag.